airbyte-cdk/bulk/core/load/src/testFixtures/kotlin/io/airbyte/cdk/load/component/TableOperationsFixtures.kt

```diff
@@ -342,12 +401,21 @@ object TableOperationsFixtures {
         return map { record -> record.mapKeys { (k, _) -> totalMapping.originalName(k) ?: k } }
     }
 
+    fun <V> List<Map<String, V>>.removeNulls() =
+        this.map { record -> record.filterValues { it != null } }
+
     suspend fun TestTableOperationsClient.insertRecords(
         table: TableName,
         records: List<Map<String, AirbyteValue>>,
         columnNameMapping: ColumnNameMapping,
     ) = insertRecords(table, records.applyColumnNameMapping(columnNameMapping))
 
+    suspend fun TestTableOperationsClient.insertRecords(
+        table: TableName,
+        columnNameMapping: ColumnNameMapping,
+        vararg records: Map<String, AirbyteValue>,
+    ) = insertRecords(table, records.toList(), columnNameMapping)
```

Comment:
Nit: Why not just make a list?

Reply:
b/c the extra level of indentation is _just_ enough to cause our formatter to add a line break :shrug: not a strong preference, but I do think the varargs invocation is slightly nicer to read

<img width="626" height="344" alt="image" src="https://github.com/user-attachments/assets/8671f7a3-db9d-4f30-895c-23d54edb9e76" />

(also, whatever our current formatter delenda est >.> )